example.secrets.dev.yml

```diff
@@ -0,0 +1 @@
+SLACK_INCOMING_WEBHOOK_URL: Go see https://api.slack.com/apps/AU7RW3DLG/incoming-webhooks?
```

Comment:
```suggestion
# Get your webhook from https://api.slack.com/apps/AU7RW3DLG/incoming-webhooks
# @example https://hooks.slack.com/services/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
SLACK_INCOMING_WEBHOOK_URL: null
```

Reply:
I agree with this change, but just to be sure, the example is not a valid webhook URL right?

Reply:
yes I always change a few characters before adding something as an @example  :-)

Reply:
This reminds me, I have to update CI as well to contain this secret, otherwise the deploy will fail

Reply:
Actually how about we fix this in a separate PR to test what notifications we get for a failing deploy on master?

Reply:
Ok for a separate PR.
We can hook the CI on the PR's branch to test the deploys, not forcibly master. What do you think ?

Reply:
We need to merge [this PR](https://github.com/gjgd/dont-slack-evil/pull/9) first @Startouf